src/sing/stateMachine.ts

```diff
@@ -91,6 +91,15 @@ type StateFactories<T extends StateDefinition[], Input, Context> = {
   ) => State<T, Input, Context> & { readonly id: U };
 };
 
+/**
+ * 初期ステートとして設定可能なステートのIDを表す型。
```

Comment:
「設定可能なステートのID」と書いていて1つの ID ではないことがわかるので、「表す型」は自明かも。
```suggestion
 * 初期ステートとして設定可能なステートのID。
```